README_BUILD.md

```diff
@@ -8,17 +8,16 @@ Make sure you have followed the [setup guide](/README_SETUP.md) before attemptin
 
 ## IMPORTANT PREREQUISITE - PLATFORM SDKs!
 
-### Using local installation
+Our build setup can find the installation of the platform SDK's for the common platforms.
+
+See the setup for the [Platform SDK](./README_SETUP.md#platform-sdk)
 
-Our build setup can find the installation the platform SDK's for these platforms
+<details><summary>Locally installed platform sdks</summary><p>
```

Comment:
I opted to minimize the wall-of-text to make it slightly more welcoming (I hope)